analytics/src/main/java/com/segment/analytics/Analytics.java

```diff
@@ -438,6 +440,9 @@ void recordScreenViews(Activity activity) {
 
   @Private
   void runOnMainThread(final IntegrationOperation operation) {
+    if (shutdown) {
```

Comment:
@stagg was wondering why this was added. This shouldn't be invoked if the shutdown code does it's job right?